tests/test_get_new_insertions.py

```diff
@@ -0,0 +1,47 @@
+import json
+import datetime as dt
+from pathlib import Path
+from urllib.parse import urljoin
+
+import pytest
+import requests
+
+import pyniva
+
+TEST_DATA_DIR = Path(__file__).parents[0] / 'test_data'
+TSB_URL = 'http://localhost:8504'
+META_URL = 'http://localhost:8506'
```

Comment:
We should test against ms ferrybox and not tsb/metaflow. This is what end users do

Reply:
good point. I will add a new test marker that requires a valid ms-ferrybox token.